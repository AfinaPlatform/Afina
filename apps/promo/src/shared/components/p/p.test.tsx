import P from "./p";
import { render } from "@testing-library/react";

describe("shared/p", () => {
  it("render p html tag", () => {
    const { container } = render(
      <P>Occaecat ullamco elit veniam commodo officia.</P>
    );
    expect(container.querySelector("p")?.tagName).toBe("P");
  });

  it("snapshot", () => {
    const { asFragment } = render(
      <P>
        Amet duis ea sint anim labore aliqua exercitation laboris officia elit
        Lorem cupidatat ad culpa.
      </P>
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
