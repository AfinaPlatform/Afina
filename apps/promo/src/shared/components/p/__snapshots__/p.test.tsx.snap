// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shared/p snapshot 1`] = `
<DocumentFragment>
  <p
    class="root"
  >
    Amet duis ea sint anim labore aliqua exercitation laboris officia elit Lorem cupidatat ad culpa.
  </p>
</DocumentFragment>
`;
